### PR TITLE
Release workflow

### PR DESCRIPTION
**Pull request by StrahinjaJacimovic**
Improvements:

- Added release asset workflow
  - Triggered on every release action
  - Added script to be used with the action

### CODE REVIEW

**Review by MarkoZiv — APPROVED** [review]
reviewed

**Review by IvanRuzavin — DISMISSED** [review]
reviewed and approved

File: scripts/index.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            doc = {
+                "name": name_without_extension,
+                "version" : version,
+                "display_name" : "NECTO project templates",
+                "hidden" : True,
+                "vendor" : "MIKROE",
+                "type" : "application",
+                "location" : f"necto/%os_type%/{asset['name']}.7z",
+                "install_location" : "%APPLICATION_DATA_DIR%",
+                "dependencies": []
```
Comment (LukicMMikroe) [inline scripts/index.py:97]:
```suggestion
                "package_changed": True
```

File: scripts/index.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            }
+        elif 'templates' == name_without_extension:
+            doc = {
+                "name": name_without_extension,
+                "version" : version,
+                "display_name" : "NECTO project templates",
+                "hidden" : True,
+                "vendor" : "MIKROE",
+                "type" : "application",
+                "location" : f"necto/%os_type%/{asset['name']}.7z",
```
Comment (LukicMMikroe) [inline scripts/index.py:95]:
```suggestion
                "download_link" : asset['url'],
```

File: scripts/index.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                "install_location" : "%APPLICATION_DATA_DIR%",
```
Comment (LukicMMikroe) [inline scripts/index.py:96]:
```suggestion
                "install_location" : "%APPLICATION_DATA_DIR%/templates",
```

**Review by IvanRuzavin — APPROVED** [review]
Reviewed and approved

**Review by MarkoZiv — APPROVED** [review]

**Pull request merged**